### DISCUSSION TITLE
Introducing EddieHub Modules for YouTube content

### DISCUSSION THREAD

**Discussion by vinzvinci** [EddieHubCommunity/support#1710] [Ideas] [answered]
Hey hey!

I think this is a great idea to start when new members join. Hmm, I know some folks prefer watching videos than reading the text right? So here's my suggestion for your YouTube content @eddiejaoude. Make it a playlist, and make it `unlisted`, so only EddieHub members that has a link can see the modules. 

`Per module = new speaker:`
ex. module 1 is assigned for @eddiejaoude, module 2 is assigned to any members who would like to volunteer for the specific module. 

Here's the <a href="https://github.com/EddieHubCommunity/support/discussions/1710#discussioncomment-818147">modules</a>

We can let <a href="https://github.com/EddieHubCommunity/support/discussions/1779">Open Source Ambassador</a> to speak in this module.

<hr />
Note: Don't make the video so long, less than 10mins or 5mins depends on the module topic

**Comment by vinzvinci** [comment]
We should include community guidelines in the module? like `NO DM`, ask permission first in the community before dm anyone, so we know what is the purpose of that person.

**Reply by eddiejaoude** [reply]
Including these "etiquette" and EddieHub community rules is a good idea 👍

**Reply by Akshu-on-github** [reply]
I think that we could utilize this to help newcomers use inclusive language too!

**Reply by vinzvinci** [reply]
We should tackle the Community roles too? Some new folks don't know about it, they are confused why other folks can share links except GitHub xD.

**Reply by schmelto** [reply]
> We should tackle the Community roles too? Some new folks don't know about it, they are confused why other folks can share links except GitHub xD. 

I think this would be a great idea!

**Reply by kaiwalyakoparkar** [reply]
I love the idea! I would love to volunteer too :)

**Comment by vinzvinci** [comment]
Public or unlisted? Comment below
![image](https://user-images.githubusercontent.com/73097560/118994842-3f7a4c80-b9b9-11eb-8c8e-67886b41ae38.png)

**Reply by iason9** [reply]
i don't think it is a good idea to put them public as the youtube channel of @eddiejaoude is actually personal and he can't just accept anyone to just upload a video to his chanel as it is branded with his own name plus he needs to grow it and putting low quality videos may do it wrong, so I think making the videos unlisted is better, the only time that public videos are better is if the quality of videos are equally or even better than Eddie's and that's another topic.

**Reply by verma-kunal** [reply]
Yeah! I agree with you on that point !!

**Reply by lainq** [reply]
Yeah, I prefer unlisted

**Reply by kaiwalyakoparkar** [reply]
> i don't think it is a good idea to put them public as the youtube channel of @eddiejaoude is actually personal and he can't just accept anyone to just upload a video to his chanel as it is branded with his own name plus he needs to grow it and putting low quality videos may do it wrong, so I think making the videos unlisted is better, the only time that public videos are better is if the quality of videos are equally or even better than Eddie's and that's another topic.

Great points. I agree on that! :+1:

**Comment by verma-kunal** [comment]
How about a topic like **" How EddieHub provides a learning environment for everyone ? "** 
I think this is an important aspect to be covered especially for the new folks who join our community and want to know how we all grow together!!
Let me know some suggestions or feedbacks as well 😃

**Reply by vinzvinci** [reply]
Great topic, I think this can include in Module 1?

**Reply by kaiwalyakoparkar** [reply]
Yup module 1 would be good place for it

**Reply by ghost** [reply]
My words are that we should not keep it into Eddie's channel, Like Eddie has a shorts channel maybe there can be an EddieHubCommunity channel. Moreover, I don't think Eddie will have a lot of time to do his stuff If he uploads the videos coming to him! We can have a community email and a community channel, this will allow only moderators/kind people you suggest to upload the videos! This will save Eddie a total of 100 hours every month!

**Comment by Akshu-on-github** [comment]
Hmm, here's a rough layout of what I've read + a few things that came to mind
- *Module 1*: All about EddieHub
- *Module 2*: Community Events (A link to schedule in the description maybe?)
- *Module 3*: Here's how we're an Inclusive Community
- *Module 4*: All things Git (videos in order of difficulty/tags with levels 1 to n)
- *Module 5*: All things GitHub (links to example repos maybe?)
- *Module 6*: FAQs (from `how do I make my first contribution?` to `why doesn't xyz work?`
...
- *Module n*: Bloopers

**Reply by vinzvinci** [reply]
Awesome, we can finalize the title of each module for the YouTube content soon. Hmm, I think about Git and GitHub is for general topics. We will only focus on EddieHub Community.

**Reply by vinzvinci** [reply]
In module 1: All about EddieHub is too big for me. Maybe: **Intro to EddieHub Community** is the best title? Then in the other module, we will talk about other activities of EddieHub.

**Reply by kaiwalyakoparkar** [reply]
Awesome! Great efforts. There are some neat points we can address

**Reply by vinzvinci** [reply]
We can also introduce this program <a href="https://github.com/EddieHubCommunity/support/discussions/1779#discussioncomment-814018">Open Source Ambassador</a>

**Reply by kaiwalyakoparkar** [reply]
> We can also introduce this program [Open Source Ambassador](https://github.com/EddieHubCommunity/support/discussions/1779#discussioncomment-814018)

Haha! Thanks for linking :)

**Reply by Akshu-on-github** [reply]
> We can also introduce this program [Open Source Ambassador](https://github.com/EddieHubCommunity/support/discussions/1779#discussioncomment-814018)

Sounds good!

**Comment by vinzvinci** [comment, marked as answer]
### Module 1 - Getting started with EddieHub
- Introduction
- What we do?
- Start your journey here to boost your career development

### Module 2 - Community Guide
- How to join the GitHub organization
- How to get assignable roles in the discord server
- How can I sign up for the monthly newsletter
- Who are and how to become our Community partner?

### Module 3 - Community rules
- Etiquette
- No DMs, spamming, and etc.
- Discord bot (EddieBot and NightBot)

### Module 4 - Here's how we're an Inclusive Community
- What is an Inclusive community means?
- Why we choose to be inclusive?

### Module 5 - Community Event
- Public speaking every weekend
- Community call every month
- MLH guild

### Module 6 - <a href="https://github.com/EddieHubCommunity/support/discussions/1779#discussioncomment-814018">Open Source Ambassador</a>
- What is the program?
- How to apply?
- Benefits

### Module 7 - Start your journey on Open Source
- How to make your first contributions?
- Why contribute to open-source?
- Benefits

<hr />
Any suggestion? what to add? what is the best title for each module? Comment below

**Reply by Akshu-on-github** [reply]
What about FAQs/short demos? Some examples include, `How do I join the GitHub Organization`, `How do I get a self assignable role` and `How can I sign up for the monthly newsletter`

**Reply by Akshu-on-github** [reply]
We could link it in the website and the server's #readme channel too

**Reply by vinzvinci** [reply]
Thanks for bringing that up. That will be cover in the Community guide - Module 1

**Reply by Akshu-on-github** [reply]
Sounds good!

**Reply by Akshu-on-github** [reply]
Oh, but depending on the length, we might want to create a separate module - maybe a `bonus` one?

**Reply by vinzvinci** [reply]
I separated the community guide

**Reply by kaiwalyakoparkar** [reply]
Great points @vinzvinci

**Reply by eddiejaoude** [reply]
This is really interesting and great 👍 - we are currently planning for `EddieCon 0.1`, but will be on this afterwards. I can get my video editor to edit the community videos too, so it is less work for the community and also keeps the videos consistent 🤓
